Latest `@octokit/auth-app` not compatible with `octokit-auth-probot`

I'm getting the following compilation error when I attempt to update the dependencies of a library that uses `octokit-auth-probot`:

```shell
node_modules/octokit-auth-probot/dist-types/types.d.ts:14:75 - error TS2694: Namespace '"/Users/bencoe/google/repo-automation-bots/packages/gcf-utils/node_modules/@octokit/auth-app/dist-types/index"' has no exported member 'AuthOptions'.

14 export declare type AuthOptions = TokenAuthTypes["AuthOptions"] | AppAuth.AuthOptions | {
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: "12.x"
       - run: npm ci

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -36,3 +36,4 @@ jobs:
       - uses: actions/setup-node@v2
       - run: npm ci
       - run: npm run lint
+      - run: npm run build

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -20,7 +20,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: "Use Node.js ${{ matrix.node_version }}"
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: "${{ matrix.node_version }}"
       - run: npm ci
@@ -33,6 +33,6 @@ jobs:
     needs: test_matrix
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
       - run: npm ci
       - run: npm run lint

--- a/.github/workflows/update-prettier.yml
+++ b/.github/workflows/update-prettier.yml
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
       - run: npm ci
       - run: "npm run lint:fix"
       - uses: gr2m/create-or-update-pull-request-action@v1.x

--- a/src/types.ts
+++ b/src/types.ts
@@ -13,7 +13,7 @@ export type StrategyOptions =
 
 export type AuthOptions =
   | TokenAuthTypes["AuthOptions"]
-  | AppAuth.AuthOptions
+  | AppAuth.InstallationAuthOptions
   | {
       type: "event-octokit";
       event: {

